doc/parameters.md

```diff
@@ -51,6 +51,25 @@ Double colon (`:`) separated list of environments (or paths) APIcast should load
 It can be used instead of `-e` or `---environment` parameter on the CLI and for example
 stored in the container image as default environment. Any value passed on the CLI overrides this variable.
 
+### `APICAST_LOAD_SERVICES_WHEN_NEEDED`
```

Comment:
I wanted to include "lazy" or "lazily" in the name of the ENV. However, I thought that would be a bit confusing because `APICAST_CONFIGURATION_LOADER` already accepts `lazy` as an option. I am open to suggestions for better names :smile:

Reply:
Some ideas:

1. APICAST_LAZY_LOADING
2. APICAST_LAZY_LOAD_SERVICES
3. APICAST_LAZY_LOAD_CONFIG

@davidor Do you have guidelines or any restrictions when creating env var names?